css/src/_score_icon.scss

```diff
@@ -0,0 +1,29 @@
+.wpseo-score-icon {
+  display: inline-block !important;
+  width: 12px !important;
+  height: 12px !important;
+  border-radius: 50% !important;
+  margin: 3px 10px 0 3px;
```

Comment:
I think this margin is actually overridden by the one on `.adminbar-seo-score` which uses !important see `margin: 10px 0 0 4px !important`

Reply:
That is only applicable to the actual adminbar, the icons are used on more places (like in the post-overview-list in the columns.